feat(developer): LDML keyboard compiler - infrastructure 🙀 (#7101)
keymanapp/keyman: developer/src/kmldmlc/README.md

Keyman Developer - LDML Keyboard Compiler
================

This package provides the following Keyman **command line tool**:

 - `kmldmlc` — takes **LDML Keyboard .xml sources** and compiles them in to a
   KMXPlus **.kmx** file.

Install
-------

Install `kmlmdlc` globally:

    npm install -g @keymanapp/ldml-keyboard-compiler

Usage
-----

> To compile a lexical model from its `.xml` source, use `kmldmlc`:

Reviewer: ```suggestion
To compile an LDML keyboard from its `.xml` source, use `kmldmlc`:
```
